net/transmission/Makefile

```diff
@@ -94,7 +93,7 @@ endef
 define Package/transmission-web
   $(call Package/transmission/template)
   TITLE+= (webinterface)
-  DEPENDS:=transmission-daemon
+  DEPENDS:=transmission-daemon-openssl||PACKAGE_transmission-daemon-polarssl
```

Comment:
The dependency should be transmission-daemon. The -openssl and -polarssl packages should use PROVIDES:=transmission-daemon